app/src/app/map/Map.module.scss

```
@@ -0,0 +1,14 @@
+@import "../../theme/base";
```

```suggestion
@import "src/theme/base";
```

We have aliases already, try merging latest `master`